src/courtroom.cpp

```diff
@@ -3995,127 +3995,7 @@ void Courtroom::on_ooc_return_pressed()
 {
   QString ooc_message = ui_ooc_chat_message->text();
 
-  if (ooc_message.startsWith("/pos")) {
-    if (ooc_message == "/pos jud") {
-      toggle_judge_buttons(true);
-    }
-    else {
-      toggle_judge_buttons(false);
-    }
-  }
```

Comment:
this looks kinda important chief

Reply:
It ain't tho. Works perfectly fine on Vanilla and Akashi without it.

Wit:
![image](https://user-images.githubusercontent.com/62221668/134748664-8a617c11-88ba-4c9b-af57-163a01ee3201.png)

Jud:
![image](https://user-images.githubusercontent.com/62221668/134748696-80bd8640-5e28-4738-a0d3-f74be5cbeb80.png)

Reply:
oh, because the [Set Position](https://github.com/AttorneyOnline/docs/blob/master/docs/development/network.md#set-position) packet was a feature added in 2.8 meant for exactly this. ok, i'm fine with this